### PR TITLE
Allow unnerfing elemental reactor

### PR DESCRIPTION
**Pull request by Seggan**
## Short Description

As per your suggestion. If accepted I will remove the code in SFW. Untested.

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.
- [x] I followed the existing code standards and didn't mess up the formatting.

### CODE REVIEW

File: src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java [hunk 1]
```diff
@@ -97,17 +97,14 @@ private void nerfCrap() {
         Reflections.setField(SlimefunItem.getById("INFINITE_PANEL"), "generation", 20_000);
         Reflections.setField(SlimefunItem.getById("INFINITY_REACTOR"), "gen", 50_000);
 
-        // SlimefunWarfare - Halved all values
-        Reflections.setField(SlimefunItem.getById("ELEMENTAL_REACTOR"), "energyProducedPerTick", 8_192);
+        if (getConfig().getBoolean("options.nerf-elemental-reactor", true)) {
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java]:
```suggestion
        if (getConfig().getBoolean("options.nerf-SlimefunWarfare", true)) {
```

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

**Comment by J3fftw1** [comment]
remove the code from your addon and this will be merged

**Review by J3fftw1 — CHANGES_REQUESTED** [review]
.

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 options:
   auto-update: true
-  nerf-other-addons: true
\ No newline at end of file
+  nerf-other-addons: true
+  nerf-SlimefunWarfare: true
```
Comment (J3fftw1) [inline src/main/resources/config.yml]: make it so if nerf-other-addons is true it also nerfs your addon

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

**Comment by WalshyDev** [comment]
Hard pass.

**Pull request closed without merging**